code/datums/recipe.dm

```diff
@@ -54,14 +54,26 @@ var/global/list/microwave_accepts_items =    list(
 	if(length(container_contents) < length(fruit))
 		return FALSE
 	var/list/needed_fruits = fruit.Copy()
-	for(var/obj/item/chems/food/snacks/grown/G in container_contents)
-		var/ktag = G.seed?.kitchen_tag
-		if(needed_fruits[ktag] > 0)
-			needed_fruits[ktag]--
+	for(var/obj/item/chems/food/snacks/S in container_contents)
+		var/use_tag
+		if(istype(S, /obj/item/chems/food/snacks/grown))
+			var/obj/item/chems/food/snacks/grown/G = S
+			if(!G.seed || !G.seed.kitchen_tag)
+				continue
+			use_tag = G.dry ? "dried [G.seed.kitchen_tag]" : G.seed.kitchen_tag
+		else if(istype(S, /obj/item/chems/food/snacks/fruit_slice))
+			var/obj/item/chems/food/snacks/fruit_slice/FS = S
+			if(!FS.seed || !FS.seed.kitchen_tag)
+				continue
+			use_tag = "[FS.seed.kitchen_tag] slice"
+		use_tag = "[S.dry ? "dried " : ""][use_tag]"
+		if(isnull(needed_fruits[use_tag]))
+			continue
+		needed_fruits[use_tag]--
 	for(var/ktag in needed_fruits)
 		if(needed_fruits[ktag] > 0)
 			return FALSE
-	return TRUE
+	return .
```

Comment:
This is redundant.

Reply:
Also you never seem to set `.` in this proc, making this nonfunctional.

Reply:
copypasta moment 😔 
```suggestion
	return TRUE
```